includes/admin/payments/class-payments-table.php

```diff
@@ -528,7 +528,9 @@ public function column_number( $order ) {
 		// View URL
 		$view_url = edd_get_admin_url( array(
 			'page' => 'edd-payment-history',
-			'view' => 'view-order-details',
+			'view' => 'order' === $order->type
```

Comment:
This is breaking for me. All completed orders are using `view-refund-details`.

My orders all habe a `type` of `sale` not `order`.

Reply:
Oops, nice catch. Fixed.